.github/ISSUE_TEMPLATE/page-request.yml

```diff
@@ -25,16 +23,7 @@ body:
         - Android
         - macOS
         - Windows
-        - General
         - Other
+        - More than one operating system
     validations:
       required: true
-  - type: input
-    attributes:
-      label: OS details
-      description: Describe any details related to an operating system.
```

Comment:
I think this field should be used to include one example usage of the command. no?

Reply:
Sounds like a good idea, but it should be optional